UndertaleModLib/Models/UndertaleEmbeddedTexture.cs

```diff
@@ -68,13 +71,13 @@ public TexData TextureData
     /// <summary>
     /// Helper variable for whether or not this texture is to be stored externally or not.
     /// </summary>
-    public bool TextureExternal { get; set; } = false;
+    internal bool TextureExternal { get; set; }
 
 
     /// <summary>
     /// Helper variable for whether or not a texture was loaded yet.
     /// </summary>
-    public bool TextureLoaded { get; set; } = false;
+    internal bool TextureLoaded { get; set; }
```

Comment:
I think this will break the "Warning: Texture failed to load!" warning in the embedded texture editor.
See here - [UndertaleEmbeddedTextureEditor.xaml, line 47-54](https://github.com/krzys-h/UndertaleModTool/blob/fc29c73a55f9212f2cc9bd948c624cf57ddcbe6b/UndertaleModTool/Editors/UndertaleEmbeddedTextureEditor.xaml#L47-L54)

Reply:
yeah probably, mustve missed that in my search results :/

Reply:
It's just within XAML, so it's not displayed in the references list